### ISSUE TITLE
Max Height

### ISSUE THREAD

**Issue by robwise** [AllenFang/react-bootstrap-table#191]
Hello, I love the gem as it's very easy to use. I have a use case where our data is dynamic, so the same table could sometimes be so large that it needs to stop getting larger and start using a scroll bar, or it could have just a single value. If I set a fixed height, then the table is too tall when I have just a couple of values. So essentially what I'm wanting to do is set a max height. Are there any plans to do this? Do you have any ideas for how to implement this and then I could try to contribute?

**Comment by AllenFang** [comment]
Hi @robwise, set a max heigh on <code>react-bootstral-table</code> is not supported, I've no idea about this requirement, so feel free to do any contribution or discuss more detail with me :)

**Comment by robwise** [comment]
@AllenFang One thing I'm doing right now to sort of fake it is something like this:

```es6
const MAX_HEIGHT = 600;
const ROW_HEIGHT = 42;
const data = [...];

return() {
  <BootstrapTable
    data={data}
    height={String(Math.min([600, (data.length + 1) * ROW_HEIGHT]))}
  >
    ...
}
```

However this seems to be unreliable because the ROW_HEIGHT can be different depending on the styling/content.

**Comment by AllenFang** [comment]
@robwise, great, I think it can be a workaround for some specified styling table.

**Comment by robwise** [comment]
@AllenFang what about adding a `maxHeight` prop, then putting some sort logic here that is similar to the above that checks for the maxHeight prop: https://github.com/AllenFang/react-bootstrap-table/blob/master/src/TableBody.js#L267

**Comment by AllenFang** [comment]
@robwise, thanks :)  I will try to use <code>max-height</code> to set the table max height, if overrflow, the scroll bar will appear.

**Comment by AllenFang** [comment]
Released on <code>v1.4.0</code>
